### PR TITLE
Adding a new TestGrid dashboard tab for conformance tests results

### LINKED ISSUE #162: Need a testgrid tab dedicated to conformance tests
It should show overall conformance tests passing rate as well as individual test results.
Right now, it's shown under continuous/release together with unit tests and E2E tests.

### PR DESCRIPTION
**Pull request by dushyanthsc**
PR adds a Dashboard tab for conformance tests results on https://testgrid.knative.dev/knative-serving dashboard.

Note: The current implementation has a limitation that it doesn't provide the summary "Overall" row. 

Fixes #162

### CODE REVIEW

File: ci/testgrid/config.yaml [hunk 1]
```diff
@@ -135,6 +135,9 @@ dashboards:
     test_group_name: ci-knative-serving-api-coverage
     description: 'Conformance tests API coverage.'
     base_options: 'exclude-filter-by-regex=Overall$&group-by-directory=&expand-groups=&sort-by-name='
+  - name: conformance
```
Comment (srinivashegde86) [inline ci/testgrid/config.yaml]: should we call this conformance-tests?

Comment (dushyanthsc) [reply ci/testgrid/config.yaml]: I don't have a preference, I will wait for others to comment to make sure there is a consensus, before making a change.

Comment (jessiezcc) [reply ci/testgrid/config.yaml]: conformance-tests sounds good to me

Comment (jessiezcc) [reply ci/testgrid/config.yaml]: and are we taking conformance out of existing "continuous" tab?

Comment (dushyanthsc) [reply ci/testgrid/config.yaml]: This PR doesn't do that, but can be done. My sense was that continuous would display results of all the tests that were run as part of continuous. Would you prefer taking them out with an exclude clause?

Comment (adrcunha) [reply ci/testgrid/config.yaml]: I prefer not removing them from the CI tab. That's the canonical way of checking that everything is green, and our "green" concept is based on looking at all the tests in it.

Comment (dushyanthsc) [reply ci/testgrid/config.yaml]: Changed tab to "conformance-tests"

**Pull request merged**